consumer/pdata/trace.go

```diff
@@ -168,7 +168,7 @@ func (sc StatusCode) String() string { return otlptrace.Status_StatusCode(sc).St
 //
 // Important: This causes a runtime error if IsNil() returns "true".
 func (ms SpanStatus) SetCode(v StatusCode) {
```

Comment:
>// Important: This causes a runtime error if IsNil() returns "true".

Do we still have IsNil() for this struct?

Reply:
Done.